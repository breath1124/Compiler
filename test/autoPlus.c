int main()
{
    int a = 2;
    a++;
    int b = 5;
    --b;
    int c = 0;
    c++;
    int d = 10;
    d--;
    print a;
    print b;
    print c;
    print d;
}