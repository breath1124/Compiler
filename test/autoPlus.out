24 19 0 5 25 13 0 2 12 13 11 13 13 11 0 1 1 12 15 -2 13 0 1 1 0 5 12 13 0 1 1 11 0 1 2 15 -1 13 0 1 1 13 0 1 1 11 0 1 2 12 15 -1 13 0 2 1 0 0 12 13 0 2 1 11 13 0 2 1 13 0 2 1 11 0 1 1 12 15 -2 13 0 3 1 0 10 12 13 0 3 1 11 13 0 3 1 13 0 3 1 11 0 1 2 12 15 -2 13 11 22 15 -1 13 0 1 1 11 22 15 -1 13 0 2 1 11 22 15 -1 13 0 3 1 11 22 21 4